source/utilities.tex

```diff
@@ -15528,6 +15528,17 @@
   template<class charT, class... Args>
     struct basic_format_string;
 
+  template<class charT> struct @\exposid{runtime-format-string}@ {                  // \expos
+  private:
+    basic_string_view<charT> @\exposid{str}@;                                       // \expos
+  public:
+    @\exposid{runtime-format-string}@(basic_string_view<charT> s) noexcept : @\exposid{str}@(s) {}
+    @\exposid{runtime-format-string}@(const @\exposid{runtime-format-string}@&) = delete;
+    const @\exposid{runtime-format-string}@& operator=(const @\exposid{runtime-format-string}@&) = delete;
```

Comment:
While implementing this paper I noticed the `const` qualifier for the return type. Can it be removed editorial?

Reply:
In general, no, but this is a deleted function, so it doesn't matter.

@burblebee , could you please remove the "const" here (in a separate commit)?

Reply:
Thanks! (I wouldn't have asked if the function wasn't deleted.)

Reply:
done :)